packages/flutter/lib/src/material/typography.dart

```diff
@@ -255,10 +283,10 @@ class TextTheme {
 
 /// The two material design text themes.
 ///
-/// [Typography.black] and [Typography.white] define the two text themes used in
-/// material design. The black text theme, which uses dark glyphs, is used on
-/// light backgrounds in light themes. The white text theme, which uses light
-/// glyphs, is used in dark themes and on dark backgrounds in in light themes.
+/// [black] and [white] define the two text themes used in material design. The
```

Comment:
sentences should start with a capital letter. To make that work here, you can write it as something like `The [black] and [white] properties provide...`